tools/api-client/python/lib/bmutils.py

```diff
@@ -98,10 +100,40 @@ def wrap_create_game(self, pbutton, obutton='', opponent=''):
     return retval.data
 
   def wrap_load_game_data(self, game):
-    retval = self.load_game_data(game)
-    if not retval.status == 'ok':
-      raise ValueError, "Failed to call loadGameData, got: " + retval
-    data = retval.data
+    # if we're using a cache directory
+    if self.cachedir:
+      # if the cache directory doesn't exist, create it
+      if not os.path.isdir(self.cachedir):
+        os.makedirs(self.cachedir)
+      # set the path to the cache file for this game
+      cachefile = self.cachedir + "/{0:010d}.json".format(game)
```

Comment:
Indeed, sadly:
```python
$ ./game_info -c ~/games/buttonmen/dev-2012/python/.bmrc -s dev 1000
Traceback (most recent call last):
  File "./game_info", line 48, in <module>
    gamedata = b.wrap_load_game_data(args[0])
  File "lib/bmutils.py", line 109, in wrap_load_game_data
    cachefile = self.cachedir + "/{0:010d}.json".format(game)
ValueError: Unknown format code 'd' for object of type 'str'
```

I don't have a strong preference between having `wrap_load_game_data()` cast its argument to an int as its first action, vs declaring that that argument needs to be an int and updating the `game_info` example to do the cast, but you have to do one of those.